server/config/mode.go

```diff
@@ -0,0 +1,84 @@
+package config
+
+import (
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/tikv/pd/pkg/errs"
+	"github.com/tikv/pd/server/storage/endpoint"
+	"go.uber.org/zap"
+)
+
+// Scheduling mode
+const (
+	Normal  = "normal"
+	Suspend = "suspend"
+	Scaling = "scaling"
```

Comment:
Does it to distinguish the scale-in or scale-out?Can all the states switch to any state?

Reply:
> Does it to distinguish the scale-in or scale-out?

No need for now perhaps. 

> Can all the states switch to any state?

I think yes. WDYT?